### PR TITLE
feat(config): change the default of useDisplayNameInClassName

### PR DESCRIPTION
**Pull request by oalbacha**
**What**: 

- Added a test to `useDisplayNameInClassName`

- Updated the FAQs with default 

- Updated the `README.md` file docs for `useDisplayNameInClassName` to say "Defaults to process.env.NODE_ENV !== 'test'"

**Why**: Not sure

**How**: Not sure

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 node_modules
+package-lock.json
```
Comment (kentcdodds) [inline .gitignore]: Thanks for this :+1:

File: src/create-glamorous.js [hunk 2]
```diff
@@ -10,9 +10,7 @@ import getGlamorClassName from './get-glamor-classname'
 export default createGlamorous
 
 function createGlamorous(splitProps) {
-  // TODO: in a breaking version, make this default to true
-  glamorous.config = {useDisplayNameInClassName: false}
-
+  glamorous.config = {useDisplayNameInClassName: process.env.NODE_ENV !== 'test'}
```
Comment (kentcdodds) [inline src/create-glamorous.js]: Perfect 👍

File: .gitignore (hunk 1 above)
Comment (oalbacha) [inline .gitignore]: Thanks for the opportunity :)

**Comment by kentcdodds** [comment]
I've changed this PR to merge into the `next` branch. Considering it's a breaking change, I want to queue up a few breaking changes to release them all at once (when a PR is merged into `master` it is automatically released). So this'll go into the `next` branch, then it'll get merged into `master` when we're ready to release this officially. Thanks again!

**Comment by kentcdodds** [comment]
Sorry, I kinda messed things up. I'll fix it :)

**Comment by kentcdodds** [comment]
I'm not sure why travis isn't kicking off another build here, but I'm going to go ahead and merge this.

Thanks again @oalbacha! Congratulations on your first ever pull request!

**Pull request merged**